AppCenterCrashes/AppCenterCrashesTests/MSACWrapperExceptionManagerTests.m

```diff
@@ -116,13 +118,45 @@ - (void)testSaveWrapperExceptionAndCrashReportWhenDirectoryWasNotCreated {
   OCMStub([mockCrashReporter crashReportPath]).andReturn(mockPath);
   OCMStub([mockCrashReporter generateLiveReport]).andReturn(mockData);
 
+  // When.
+  [MSACWrapperExceptionManager setCrashReporter:mockCrashReporter];
+  MSACWrapperException *wrapperException = [self getWrapperException];
+
+  // Then.
+  OCMReject([mockUtility createFileAtPath:OCMOCK_ANY contents:OCMOCK_ANY attributes:OCMOCK_ANY]);
+
+  // When.
+  [MSACWrapperExceptionManager saveWrapperExceptionAndCrashReport:wrapperException];
+
+  // Stop mocking.
+  [mockUtility stopMocking];
+  [mockCrashReporter stopMocking];
+}
+
+- (void)testSaveWrapperExceptionAndCrashReportWhenFailedToCreateFile {
+
+  // If.
+  id mockUtility = OCMClassMock([MSACUtility class]);
+  OCMStub(ClassMethod([mockUtility createDirectoryAtURL:OCMOCK_ANY])).andReturn(YES);
+  OCMStub(ClassMethod([mockUtility createFileAtPath:OCMOCK_ANY contents:OCMOCK_ANY attributes:OCMOCK_ANY])).andReturn(NO);
+
+  // Mock crashReporter.
+  id mockCrashReporter = OCMClassMock([PLCrashReporter class]);
+  NSString *mockPath = @"file://mock/live_report.plcrash";
+  NSData *mockData = [NSData new];
+  OCMStub([mockCrashReporter crashReportPath]).andReturn(mockPath);
+  OCMStub([mockCrashReporter generateLiveReport]).andReturn(mockData);
+
+  // Remove file if it exists
+  [[NSFileManager defaultManager] removeItemAtPath:mockPath error:nil];
+
   // When.
   [MSACWrapperExceptionManager setCrashReporter:mockCrashReporter];
   MSACWrapperException *wrapperException = [self getWrapperException];
   [MSACWrapperExceptionManager saveWrapperExceptionAndCrashReport:wrapperException];
 
   // Then.
-  OCMReject([mockUtility createFileAtPathComponent:OCMOCK_ANY withData:OCMOCK_ANY atomically:YES forceOverwrite:YES]);
+  XCTAssertFalse([[NSFileManager defaultManager] fileExistsAtPath:mockPath]);
```

Comment:
Could you please also update the changelog about these [changes](https://github.com/microsoft/appcenter-sdk-apple/pull/2380) here?